src/common/components/vanilla/heading/heading.css

```diff
@@ -40,3 +40,7 @@
   font-size: 1.063rem;
   line-height: 1.412;
 }
+
+.heading a {
+  color: inherit;
```

Comment:
Making link in headers in text colour so we don't end up with huge orange titles.